server/server.js

```diff
@@ -68,6 +69,16 @@ app.get(`${apiBaseUrl}/participants`,
   asyncMiddleware(async (req, res) => {
     const user = req.hcapUserInfo;
     const result = await getParticipants(user);
+    const { content } = req.kauth.grant.access_token;
+    logger.info({
+      "action": "participant_get",
+      "performed_by": {
+        "username": content.preferred_username,
```

Comment:
This may be considered PII and should be kept out of logs. We don't even really need to use `content` because we are using the `keycloak.getUserInfoMiddleware` middleware with the results assigned to the `user` var on line 70. This could be reduced to `performed_by: user.id`.